packages/mephisto-remote-query/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "mephisto-remote-query",
```

Comment:
As mentioned in our 1:1 earlier, I'm leaning towards keeping this functionality bundled with `useMephistoTask` and `useMephistoLiveTask` in  the`mephisto-task` pkg. It allows folks to easily opt-in to add'l functionality as their tasks evolve in design without installing new deps, it makes operations easier around deploying packages as well as reducing complexity around peer-dep version mismatches w/ `mephisto-task`. Since this package is also pretty lightweight and somewhat related in functionality, I think it's fine to keep it with the other code.